subworkflows/local/build_indices.nf

```diff
@@ -0,0 +1,155 @@
+//
```

Comment:
this file does not exists anymore, it was replace by prepare_genome

Reply:
ahhh, thanks